.github/workflows/CI.yml

```diff
@@ -29,4 +29,5 @@ jobs:
       run: go test -v ./...
 
     - name: Format
+      if: matrix.platform == 'macos-latest'
```

Comment:
I'd use ubuntu. it's easier to provision linux containers than windows and macos, which I would think would mean it runs faster/less issues.

Reply:
Done, but just to make sure I understand, according to https://resources.github.com/whitepapers/GitHub-Actions-Cheat-sheet/#:~:text=A%20list%20of%20the%20steps%20that,all%20on%20the%20same%20virtual%20environment. It seams that the VM setup will happen only once per other matrix dimensions for MacOS, right?

Reply:
/shrug 
I'm making assumptions based on my experience that setting up linux is easier than macOS. I could be wrong when it comes to github actions specifically.

Reply:
oh sorry for not being clear.
I meant that the difference between conditionally running the Format on Linux instead of MacOS is how fast will the gofmt command run on Linux vs MacOS and not on how fast setting a Linux VM vs setting MacOS VM because they will be set regardless for the previous steps, right?